apps/condo/domains/organization/components/Recipient/SettingsContent.tsx

```diff
@@ -0,0 +1,83 @@
+import get from 'lodash/get'
+import { useIntl } from '@condo/next/intl'
+
+import React from 'react'
+import { Row, Col, Typography, Alert } from 'antd'
+import styled from '@emotion/styled'
+import { Gutter } from 'antd/es/grid/row'
+import { colors } from '@condo/domains/common/constants/style'
+
+import { useOrganization } from '@condo/next/organization'
+import { Recipient } from '@condo/domains/organization/components/Recipient'
+
+import { BillingRecipient, BillingIntegrationOrganizationContext } from '@condo/domains/billing/utils/clientSchema'
```

Comment:
It would be good to sort imports alphabetically.

Reply:
Updated as prompted @akarjakin

Reply:
ok